lib/OpenQA/WebAPI/Controller/API/V1/Webhook.pm

```diff
@@ -131,9 +129,11 @@ sub product ($self) {
     $params{CI_TARGET_URL} = $base_url if $base_url;
 
     # set GitHub parameters so the Minion job will be able to report the status back to GitHub
+    my $html_url = $pr->{html_url};
     $params{GITHUB_REPO} = $repo_name;
     $params{GITHUB_SHA} = $sha;
     $params{GITHUB_STATUSES_URL} = $statuses_url;
+    $params{GITHUB_PR_ID} = $pr_id;
```

Comment:
Would you mind elaborating how this is helpful? It doesn't look like the ID is getting used anywhere.

Reply:
As stated in the commit message:
> * We need some unique ID to associate subsequent updates with
>    an existing scheduled product to be able to implement
>    cancellation and cleanup

I've realized that adding this to the settings might not be the best idea performance-wise and created #5133 for introducing another column.

Regardless how we do this - in this PR alone it'll be not used indeed (except for being displayed) and the code will likely change slightly later. I've been creating this PR mainly to avoid too many changes in one PR but I can also close it if incomplete changes aren't wanted.